client/ayon_core/cli.py

```diff
@@ -386,6 +386,53 @@ def main(*args, **kwargs):
 
     _cleanup_project_args()
 
+    app_addon = addons_manager.get_enabled_addon("applications")
```

Comment:
Do you realize that this cli is also used for small subprocesses like extract burnin, ffmpeg user service ocio functionality etc? Some of them are already outputing to parent process. On windows it also doesn't make sense to read output if stdout/stderr is not available at all (is set to `None`).

I don't think we should do it automatically for all processes TBH. It should be explicit and not enabled by default. Have one function in ayon-core to store the process info that can be easily called.

Reply:
With that function we should move that logic to ayon-core.